### PR TITLE
All Files except tests now have Javadocs Comments

### PR DESCRIPTION
**Pull request by honnuanand**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -255,7 +255,7 @@
                 <executions>
                     <execution>
                         <id>sign-artifacts</id>
-                        <phase>verify</phase>
+                        <phase>deploy</phase>
```
Comment (pacphi) [inline pom.xml:258]: Nice! That'll do.

File: src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/LlamaProperties.java [hunk 2]
```diff
@@ -9,12 +9,26 @@
 import java.util.List;
 import java.util.Optional;
 
+/**
+ * Sample properties aws.bedrock.model.llama2.modelId=meta.llama2-13b-chat-v1
+ * aws.bedrock.model.llama2.topP=0.9 aws.bedrock.model.llama2.temperature=0.5
```
Comment (pacphi) [inline src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/LlamaProperties.java:14]: Careful. Line break. Each key-value pair should appear on separate line for readability.

File: src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/ClaudeProperties.java [hunk 3]
```diff
@@ -9,6 +9,15 @@
 import java.util.List;
 import java.util.Optional;
 
+/**
+ * Sample properties aws.bedrock.model.claude.id=anthropic.claude-v2
+ * aws.bedrock.model.claude.prePrompt=Human:
+ * aws.bedrock.model.claude.postPrompt=Assistant: aws.bedrock.model.claude.maxTokens=100
```
Comment (pacphi) [inline src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/ClaudeProperties.java:15]: Line break. Each key-value pair should be on a separate line for readability purposes.

File: src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/JurassicProperties.java [hunk 4]
```diff
@@ -17,10 +17,20 @@
 @Validated
 @ConfigurationProperties(prefix = "aws.bedrock.model.jurassic")
 public record JurassicProperties(@Pattern(
+		/**
```
Comment (pacphi) [inline src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/JurassicProperties.java:20]: The javadoc at the class level... please have each key-value pair appear on a separate line.

File: src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/TitanProperties.java [hunk 5]
```diff
@@ -9,13 +9,29 @@
 import java.util.List;
 import java.util.Optional;
 
+/**
+ * Sample properties aws.bedrock.model.titan.id=amazon.titan-text-express-v1
+ * aws.bedrock.model.titan.temperature=0.5 aws.bedrock.model.titan.topP=0.9
+ * aws.bedrock.model.titan.maxTokens=100 aws.bedrock.model.titan.stopSequences=Human:,
+ * Assistant:
+ *
+ * @see <a href="https://docs.ai21.com/reference/titan-complete-api-ref">...</a>
```
Comment (pacphi) [inline src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/TitanProperties.java:18]: This is not a valid hyperlink.  Please fix to point to correct documentation.  Consider https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-titan.html.

File: src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/LlamaProperties.java [hunk 6]
```diff
@@ -9,12 +9,26 @@
...
+ * aws.bedrock.model.llama2.maxTokens=100
+ *
+ * @see <a href="https://docs.ai21.com/reference/llama2-complete-api-ref">...</a>
```
Comment (pacphi) [inline src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/LlamaProperties.java:17]: Not correct or valid hyperlink.  Consider https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-meta.html.

File: src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/JurassicProperties.java (hunk 4 above)
Comment (pacphi) [reply src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/JurassicProperties.java:20]: The hyperlink should updated to be `@see`  with https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-jurassic2.html.

File: src/main/java/io/clue2solve/aws/bedrock/springboot/starter/autoconfigure/aimodels/OnTitan.java [hunk 7]
```diff
@@ -3,37 +3,61 @@
 import org.springframework.boot.autoconfigure.condition.AnyNestedCondition;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 
+/**
+ * Condition class for Titan
+ */
 class OnTitan extends AnyNestedCondition {
```
Comment (pacphi) [inline src/main/java/io/clue2solve/aws/bedrock/springboot/starter/autoconfigure/aimodels/OnTitan.java:9]: Pay attention to each `On*` condition name.  Adjust the "Condition for" to match the exact Titan model variant.  Thanks!

**Review by pacphi — CHANGES_REQUESTED** [review]
Please review and implement the changes requested.

**Comment by honnuanand** [comment]
Resolved all the comments/suggestions

**Comment by honnuanand** [comment]
just commited all the changes

**Review by pacphi — APPROVED** [review]

**Pull request merged**